foxpuppet/windows.py

```diff
@@ -43,3 +47,28 @@ def window_element(self):
         """
 
         return self.selenium.find_element(By.CSS_SELECTOR, ':root')
+
+    def close(self, handle):
+        """ Closes the window with the specified handle """
+
+        self.switch_to(handle)
+
+        self.selenium.close()
+
+    def switch_to(self, target):
+        """Switches context to the specified chrome window"""
+
+        target_handle = None
+
+        # Check if requested handle is available
+        if target in self.all:
+            target_handle = target
+        else:
+            print("Handle not found")
```

Comment:
If the handle isn't there it raises a 'Browser undefined' message. Is that okay? I'd like for it to be a bit more specific.